### DISCUSSION TITLE
Are there any contribution guidelines/what am I allowed to contribute?

### DISCUSSION THREAD

**Discussion by Luiggi33** [MVladislav/ansible-cis-ubuntu-2404#3] [General]
As the title says, I just wanted to ask if you have any guidelines for this and if you have any things you feel that need the most attention

**Comment by MVladislav** [comment]
Currently, there are no formal contribution guidelines in place. However, I do have a few suggestions:

- When proposing changes, please ensure that they align with the existing structure. Any additions or modifications should maintain the project's overall organization.
- You can either create an issue for discussion, which I will review when I have time, or submit a pull request directly, and I'll verify the changes.
- **The main focus at the moment is testing and verifying.**
- Most aspects are still **in line with 22.04**, **small changes** have already been **adjusted** accordingly.
- I'm currently working to finalize the last set of changes, with the goal of releasing v1.0.0 to the main branch soon.
  - Currently, the priority is running multiple test cases to ensure all configurations work as intended, while addressing any fix points from the 22.04 ansible version.

Feel free to contribute based on the above points, and I'll assist with reviews when needed!

**Reply by Luiggi33** [reply]
Then I'll start of by some testing. I have already submitted my first issue, but apart from that the playbook ran through. I will verify these changes later and give feedback accordingly
